ocaml/xapi/xapi_pool_update.ml

```diff
@@ -170,7 +171,7 @@ let create_yum_config ~__context ~self ~url =
   let signed = String.length key <> 0 in
   let signed_index = if signed then 1 else 0 in
   let name_label = Db.Pool_update.get_name_label ~__context ~self in
-  (Printf.sprintf ("[main]\nkeepcache=0\nreposdir=/dev/null\ngpgcheck=%d\nrepo_gpgcheck=%d\n\n") signed_index signed_index)
+  (Printf.sprintf ("[main]\nkeepcache=0\nreposdir=/dev/null\ngpgcheck=%d\nrepo_gpgcheck=%d\ninstallonlypkgs=\n\n") signed_index signed_index)
```

Comment:
Suggestion for the commit message: `For pool update, generate yum.conf with installonlypkgs` or similar.